catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/builder/AttributeValueNormalizer.java

```diff
@@ -45,7 +45,7 @@
   // Primary purpose is to protect persistence layer from bad symbols leaking into valid prop names
   // https://codice.atlassian.net/wiki/spaces/CODICE/pages/77627393/Taxonomy+Guidelines
   private static final Pattern EXPECTED_ATTRIBUTE_NAME_PATTERN =
-      Pattern.compile("(ext\\.)?(\\p{Alpha}+\\.)?\\p{Alpha}+(-\\p{Alpha}+)*");
+      Pattern.compile("\\p{Alpha}+([-.]\\p{Alpha}+)*");
```

Comment:
There we go. That looks better. The regex can now be read as:
- One or more letters, upper or lower case, as a single group.
- **Optionally** followed by one or more additional groups, which must start with a delimiter (either a dot `.` or dash `-`) followed by one or more letters (upper or lower case).